pyfar/classes/audio.py

```diff
@@ -591,12 +618,19 @@ def __init__(
 
         # initialize signal specific parameters
         self._sampling_rate = sampling_rate
+
+        assert (isinstance(complex, bool))
```

Comment:
should raise a more verbose error, see above.

Reply:
I added that.